Quantum-Control-Applications/Superconducting/Single-Fixed-Transmon/set_octave.py

```diff
@@ -1,79 +1,38 @@
 """
-set_octave.py: script containing helper functions for setting all the octave parameters
+set_octave.py: script for initiating the octave
 """
-from qm.octave import *
 import os
 from qm.octave import QmOctaveConfig
-from qm.octave.octave_manager import ClockMode
-from typing import Union
 
 
 class OctaveUnit:
     """Class for keeping track of OctavesSettings in inventory."""
 
     def __init__(
-        self,
-        name: str,
-        ip: str,
-        port: int = 50,
-        clock: str = "Internal",
-        con: str = "con1",
-        port_mapping: Union[str, list] = "default",
+            self,
+            name: str,
+            ip: str,
+            port: int = 50,
```

Comment:
Maybe we should remove the default port, because with QOP220 it is not 50 anymore...

Reply:
Same, if the the Octave default IP address ends by .250, then `port: int = 11250` is good, otherwise we can just remove the default value to raise an error if the user doesn't specify it (`port: int,`)